main.py

```diff
@@ -0,0 +1,121 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import models
+import schemas
+from database import SessionLocal
+
+PAGE_SIZE = 10
+
+app = FastAPI()
```

Comment:
In the main.py file, integrate the parts created in previous steps to build the FastAPI application:

Create the database tables using the SQLAlchemy models.

Reply:
done